usr/share/regolith/sway/config.d/80_grimshot

```diff
@@ -0,0 +1,11 @@
+## Session // Window Snapshot // <><Prtscr> ##
+set_from_resource $wm.binding.screenshot.window wm.binding.screenshot.window Print
+bindsym $mod+$wm.binding.screenshot.window exec grimshot copy active
```

Comment:
Might be a good idea to add --notify here as well.

Reply:
My thinking is that this particular keybinding may be used quite often by some users, and since it doesn't mutate the filesystem, doesn't necessitate a notification.  Additionally, these would have to be cleaned up manually (or wait until reboot) for each "window shot".

Ideally though yes we'd have some subtle indication that the action has been performed, perhaps by "flashing" the window border or something.  I was looking for cli tools that might offer these effects for shell scripts but came up empty handed.